dataminer-overview/DataMiner_Solutions/Overview_DataMiner_Solutions.md

```diff
@@ -65,3 +65,8 @@ Key features:
 Continually evolving along a roadmap, the DataMiner solutions are subject to updates, ensuring compatibility with newer versions while preserving backward compatibility. Updates are facilitated by deploying updated solution packages from the DataMiner Catalog. You have full autonomy over solution implementations, so you can modify visuals, Automation scripts, and more, according to your preferences.
 
 However, keep in mind that if you modify components of a solution, these will be overwritten when you deploy an update of the solution. Consequently, if you intend to tailor solutions to your unique context, we recommend that you **create customized versions** by copying (or branching) them and then applying necessary adjustments, thereby preserving the original solution for future updates.
+
+> [!TIP]
+>
+> - The [DOM Viewer](xref:domviewer_about) is a sophisticated low-code application that enables effortless visualization and comprehensive control of your DataMiner Object Model (DOM) definitions and instances, offering intuitive node-edge graph exploration, detailed field information, state transition tracking, and extensive filtering options for both definitions and instances across all modules. 
```

Comment:
I'm not sure if this is the best place to put this info: it seems to have little to do with the "Updates" section where you placed it. However, since you added DOM Viewer as a separate solution under DataMiner Solutions, it might be more appropriate to instead add a link to it under the "Available solutions" header on this page (line 36).

Reply:
Much appreciated for the feedback.
I moved the link under the "Availabe Solutions" section as suggested with the adequate changes to the format.